packages/ui-components/src/components/windowedlist.ts

```diff
@@ -1420,6 +1587,7 @@ export namespace WindowedList {
      * Items list to be rendered
      */
     itemsList: {
+      get(index: number): T;
```

Comment:
Should be optional to be backward compatible:

```suggestion
      get?: (index: number) => T;
```